src/pages/profile/jsoneditor.ts

```diff
@@ -1,37 +1,44 @@
-import * as m from "mithril"
-import { CodeJar } from "codejar"
```

Comment:
Is CodeJar used anywhere else? I don't see it being removed from the dependency list, but it was removed from the JsonEditor file.

Reply:
No longer used; I'll push a commit to take care of that!